### PR TITLE
Port from 0.8.12 part1

### PR DESCRIPTION
**Pull request by sa2ajj**

### CODE REVIEW

File: master/buildbot/util/__init__.py [hunk 1]
```diff
@@ -209,6 +209,35 @@ def datetime2epoch(dt):
         return calendar.timegm(dt.utctimetuple())
 
 
+# TODO: maybe "merge" with formatInterval?
+def human_readable_delta(start, end):
```
Comment (tardyp) [inline master/buildbot/util/__init__.py]: I think this would belong to reporter.util as I dont see how it could be useful appart from that.

**Comment by tardyp** [comment]
:+1: good work!

File: master/buildbot/util/__init__.py (hunk 1 above)
Comment (sa2ajj) [inline master/buildbot/util/__init__.py]: I agree.

Though let's move it when `buildbot/status/github.py` moves to `buildbot/reporters/github.py`

**Comment by sa2ajj** [comment]
Looks like I managed to fix test cases (let's see what Travis says).

**Pull request merged**